decompressed/gui_file/usr/share/transformer/scripts/upgradegui

```diff
@@ -144,7 +145,7 @@ if [ ! -f $WORKING_DIR/$FILE_NAME ]; then #Check if file exist as offline upload
 	logger_command "Downloading GUI file..."
 	# Enter /tmp folder
 	if ! cd "$WORKING_DIR"; then
-	logger_command "ERROR: can't access $WORKING_DIR" >&2
+	logger_command "ERROR: Can't access $WORKING_DIR" >&2
 	ubus send fwupgrade '{"state":"done"}'
```

Comment:
Call a handler here to say fw upgrade failed? eg. ubus send fwupgrade '{"state":"failed"}'

Reply:
I prefer to leave in lowercase "can't".
For me it's better to leave untouch as much as possible the English strings in te esource.